spec/support/hyrax/basic_metadata.rb

```diff
@@ -25,6 +25,9 @@ def self.included(work)
       work.property :related_url, predicate: ::RDF::RDFS.seeAlso
       work.property :bibliographic_citation, predicate: ::RDF::Vocab::DC.bibliographicCitation
       work.property :source, predicate: ::RDF::Vocab::DC.source
+
+      # From Zizia
+      work.property :deduplication_key, predicate: 'http://metadata.example.com/vocab/predicates#deduplicationKey'
```

Comment:
I'm wondering if we just want to implement a `primary_id` and find a better predicate.  It feels like that's what most adopters are going to want anyway.  

How about [:RDF::VOCAB::BF2::identifiedBy](https://github.com/ruby-rdf/rdf-vocab/blob/6631cd4816c19ecfe0511708d023c00f8d25c2ca/lib/rdf/vocab/bf2.rb#L1723)

Reply:
I think adding a more specific predicate may ultimately be confusing because then institutions may be tempted to use it for some other purpose. Right now the the `deduplication_key` is something that's only used for grouping the `Work` and `FileSet`s during the import.

Reply:
I changed the predicate to `::RDF::VOCAB::BF2.identifiedBy`, but I really think we should keep using the name `deduplication_key` since Emory is already using that.